user_sync/config.py

```diff
@@ -84,6 +87,24 @@ def set_options(self, caller_options):
     def get_logging_config(self):
         return self.main_config.get_dict_config('logging', True)
 
+    def validate_limits_config(self):
+        ''' Ensure that we have limits set for max deletions per run and max missing users
+        '''
+        limits_config = self.main_config.get_dict_config('limits', True)
```

Comment:
I would just not allow None (main_config.get_dict_config('limits')).    The error message is a bit misleading because the section is "limits" and people could be using "Limits" to fix.